apps/bluetooth/app.js

```diff
@@ -661,6 +660,11 @@ module.exports = function (activity) {
     }
   })
 
+  activity.on('active', () => {
+    logger.log(`activity.onActive(${currentSkillName})`)
+    activity.setContextOptions({ keepAlive: true })
```

Comment:
Does need back false when `inactive`?

Reply:
It's automatically cleared in VUI on context destroys, which means options would be cleared on `destroy` and `background`.